tests/foreman/longrun/test_capsule_loadbalancer.py

```diff
@@ -0,0 +1,296 @@
+"""Test class for Capsule Loadbalancer
+
+:Requirement: Loadbalancer capsule
+
+:CaseAutomation: ManualOnly
+
+:CaseLevel: Integration
+
+:CaseComponent: Capsule
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+
+from robottelo.decorators import tier1
+
+
+@pytest.mark.stubbed
+@tier1
+def initail_setup():
```

Comment:
```suggestion
def initial_setup():
```

Please pay attention to committing typos. Our review process is long enough already, no need to prolong it additionally by something as trivial.

Reply:
My bad. I have corrected it. Thanks